sources/net/Network2.cpp

```diff
@@ -380,13 +380,13 @@ void SocketHandler::sendMessage(Message2 message)
 
     QByteArray data;
 
-    uchar temp[4];
+    char temp[4];
```

Comment:
The travis build is failing because of this change.
`qToBigEndian` takes `uchar*` as argument in old qt versions, and gcc treats `char` and `uchar` differently.
So cast here is needed unfortunately.